opencog/haskell/OpenCog/AtomSpace/Api.hs

```diff
@@ -57,51 +59,47 @@ foreign import ccall "AtomSpace_addNode"
   c_atomspace_addnode :: AtomSpaceRef
                       -> CString
                       -> CString
-                      -> Ptr UUID
+                      -> Handle
```

Comment:
Unless Handle in Haskel is always the same thing as Handle* in C++ -- because I see this everywhere in the code.  If so, the naming convention is confusing.